bundler/tool/bundler/standard_gems.rb.lock

```diff
@@ -63,4 +61,4 @@ DEPENDENCIES
   test-unit
 
 BUNDLED WITH
-   2.3.0.dev
+   2.2.24
```

Comment:
Can you update the `update_locked_bundler` rake task at the root of the repsository and run it to update all lockfiles?